src/transformers/generation/configuration_utils.py

```diff
@@ -216,6 +216,9 @@ class GenerationConfig(PushToHubMixin):
             more details.
         output_scores (`bool`, *optional*, defaults to `False`):
             Whether or not to return the prediction scores. See `scores` under returned tensors for more details.
+        output_logits (`bool`, *optional*, defaults to `False`):
```

Comment:
```suggestion
        output_logits (`bool`, *optional*):
```
(see related comment below)